packages/calcite-components/src/components/date-picker-day/date-picker-day.scss

```diff
@@ -31,7 +31,6 @@
 
 .text {
   @apply mt-px mb-0;
-  margin-inline-start: theme("spacing.px");
```

Comment:
@eriklharper can you confirm that eslint or prettier removed the correct one of these duplicate css properties?

Reply:
I blamed it and it's from this commit: https://github.com/Esri/calcite-design-system/commit/4169f30984f92f5146d1b3709a99fbe9062d6b68 

The one not being used was removed so I assume that's correct, but just wanted to double check before its gone forever.